app/main/views/services.py

```diff
@@ -157,6 +157,18 @@ def update_section(service_id, section):
                 "user",
                 "supplier app")
         except HTTPError as e:
+            errors_map = {}
+            for error in e.response.json()['error'].keys():
+                id = existing_service_content.get_question(error)['id']
```

Comment:
I get `u"Additional properties are not allowed (u'lastUpdatedByEmail', u'lastCompletedByEmail', u'lastCompleted', u'lastUpdated' were unexpected)"` which is probably related to the recent changes to the api.

But I still think we should handle the `_form` or any other unexpected key here.